.nexus-conflict-rules.yaml

```
@@ -0,0 +1,200 @@
+# Nexus项目冲突检测规则配置
+# 定义项目中各种文件冲突的检测规则和偏好设置
+
+# 项目偏好设置
+preferences:
+  # JavaScript/Node.js包管理器偏好
+  js_package_manager: "pnpm"  # 可选: npm, pnpm, yarn
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)